modules/twinklewarn.js

```diff
@@ -54,8 +67,8 @@ Twinkle.warn = function twinklewarn() {
 			}
 		};
 
-		warnFromTalk($('#mw-diff-otitle2 .mw-usertoollinks a').first());
-		warnFromTalk($('#mw-diff-ntitle2 .mw-usertoollinks a').first());
+		warnFromTalk('otitle');
```

Comment:
```suggestion
		warnFromTalk('otitle'); // older diff
```

And likewise for the latter. It's confusing to see passed nonsensical strings passed to the function, though i see there's no better alternative.